entries/p/protolabs.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Protolabs": {
+    "domain": "protolabs.com",
+    "url": "https://www.protolabs.com/",
```

Comment:
```suggestion

```
The URL field is not necessary in this case (See [CONTRIBUTING.md](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#new-sites) for more info)